iphone/Maps/UI/Settings/MWMSettingsViewController.mm

```diff
@@ -191,6 +198,28 @@ - (void)configNavigationSection
     voiceInstructions = L(@"duration_disabled");
   }
   [self.voiceInstructionsCell configWithTitle:L(@"pref_tts_language_title") info:voiceInstructions];
+
+  using namespace routing;
+
+  NSString * info = nil;
+  switch (f.GetRoutingManager().GetSpeedCamManager().GetMode())
+  {
+  case SpeedCameraManagerMode::Auto:
+    info = L(@"speedcam_option_auto");
+    break;
+  case SpeedCameraManagerMode::Always:
+    info = L(@"speedcam_option_always");
+    break;
+  case SpeedCameraManagerMode::Never:
+    info = L(@"speedcam_option_never");
+    break;
+  case SpeedCameraManagerMode::MaxValue:
+    CHECK(false, ());
```

Comment:
Please write comment for CHECK(false, ... . That means: `CHECK(false, ("Unexpected mode SpeedCameraManagerMode::MaxValue."));`

Reply:
done